flocker/node/agents/ebs.py

```diff
@@ -60,10 +69,11 @@ def _blockdevicevolume_from_ebs_volume(ebs_volume):
 
     :return: Input volume in BlockDeviceVolume format.
     """
+    ebs_volume.update()
     return BlockDeviceVolume(
         blockdevice_id=unicode(ebs_volume.id),
         size=int(GB(ebs_volume.size).to_Byte().value),
-        attached_to=None,
+        attached_to=ebs_volume.attach_data.instance_id,
```

Comment:
Yes, ran all tests except `*resize*` and `test_detach_volume`. `test_detach_volume` has a mount failure that i need to look into. Rest all tests, listed in `test_ebs.py` in this pull request pass.